### PR TITLE
[batch-request]feat: cp all header to every request

### PR DESCRIPTION
**Pull request by ShiningRush**
cp all header to every request except the headers start with "content-", keep same as [google batch request](https://developers.google.com/classroom/guides/batch)

### CODE REVIEW

File: t/plugin/batch-requests.t [hunk 1]
```diff
@@ -716,24 +722,28 @@ passed
...
-                            "X-Cookie": "request-cookies-d"
+                            "X-Cookie": "request-cookies-d",
+                            "X-HeaderD": "request-header-d"
                         }
                     }
                 ]]=],
                 {
-                    Cookie = "request-cookies"
+                    Cookie = "request-cookies",
+                    Header = "request-header"
```
Comment (membphis) [inline t/plugin/batch-requests.t:746]: that is not a good name, we'd better use a different one.

Comment (ShiningRush) [reply t/plugin/batch-requests.t:746]: You mean we should not name key like "Cookie" or "Header"?But browner will send cookie into the http header named "Cookie", it is http specification.

Comment (membphis) [reply t/plugin/batch-requests.t:746]: `Header` is not a good name. `Cookie` is fine.

**Comment by membphis** [comment]
@ShiningRush merged, many thx

**Pull request merged**